04_setup_ironic.sh

```diff
@@ -29,6 +29,14 @@ jq -s '.[0] * .[1]' ${REGISTRY_AUTH_FILE} ${REGISTRY_CREDS} > ${COMBINED_AUTH_FI
 DOCKERFILE=$(mktemp --tmpdir "release-update--XXXXXXXXXX")
 _tmpfiles="$_tmpfiles $DOCKERFILE"
 echo "FROM $OPENSHIFT_RELEASE_IMAGE" > $DOCKERFILE
+
+# To build custom images is highly recommended to build a base image first.
+# Build the base image if we set a custom repo file in the config file.
+if [[ -n ${CUSTOM_REPO_FILE:-} ]]; then
```

Comment:
What about using '-f' instead? In this way it will also check the effective presence of the file

Reply:
that's a good idea!
the main reason I didn't use -f is because I didn't want for the variable to have the absolute path of the file, but just the name; if the file doesn't exist, the process  fails anyway during image building
BUT I guess it can be useful in case we have the repositories file in another directory and not in the base-image dir
ok, I'll change it and see what happen :)